MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -449,6 +441,37 @@ - (nullable NSURL *)buildTokenRequestURLWithAppSecret:(NSString *)appSecret rele
   return components.URL;
 }
 
+- (void)openUrlInSafari:(NSURL *)url {
```

Comment:
Can we rename this to something like `startAuthenticationForUrl:(NSURL *)url`? The method name is misleading as only iOS 8 will actually open Safari. The method name confused me for a bit as I thought "where the heck did the `SFAuthenticationSession`/`SFSafariViewController` logic go?!". But definitely 👍 for extracting the logic into it's own method.

Reply:
👍 Good point! `openUrlInSafari` is kind of confusing. But I wonder if having `Authentication` in the function name would not make sense for the non-iOS 11 cases where it doesn't actually pull up that authentication message before going into the SafariViewController?